File: Question1/fizzbuzz.py
def fizzbuzz(n):
    output = ['']*(n-1)
    for i in range(1, n):
        if (i % 3 == 0 and i % 5 == 0):
            x = 'FizzBuzz'
        elif (i % 3 == 0):
            x ='Fizz'
        elif (i % 5 == 0):
            x = 'Buzz'
        else:
            x = str(i)
        output[i-1] = x
    return output

if __name__ == '__main__':
    print(fizzbuzz(15))

File: Question1/FizzBuzz.js
let testNumber = 15;
let outputArray =[]
//Output: ["1","2","Fizz","4"]

for(let i=1; i<=testNumber;i++){
    if(i%3==0&&i%5==0){
        outputArray.push("fizzbuzz")
    }
    else if(i%3==0){
        outputArray.push("Fizz")
    }
    else if(i%5==0){
        outputArray.push("Buzz")
    }
    else{
        outputArray.push(`${i}`)
    }
}
console.log(outputArray)
